pkg/eraser/eraser.go

```diff
@@ -73,8 +112,8 @@ func parseEndpoint(endpoint string) (string, string, error) {
 	}
 }
 
-func getImageClient(ctx context.Context) (pb.ImageServiceClient, *grpc.ClientConn, error) {
-	addr, dialer, err := GetAddressAndDialer("unix:///run/containerd/containerd.sock")
+func getImageClient(ctx context.Context, socketPath string) (pb.ImageServiceClient, *grpc.ClientConn, error) {
```

Comment:
Not fore a future change, we can probably have this return a `(*client, error)` instead of the image client and the client conn (although we need to pass through a `Close` call to close the conn when we are done.